jbpm-designer-backend/src/main/java/org/jbpm/designer/repository/vfs/VFSRepository.java

```diff
@@ -575,746 +599,144 @@ public void createGlobalDirOnNewProject( @Observes NewProjectEvent newProjectEve
-                    "                \"nodetype\" : \"http://b3mn.org/stencilset/bpmn2.0#IntermediateMessageEventCatching\",\n" +
-                    "                \"namespace\" : \"http://b3mn.org/stencilset/bpmn2.0#\",\n" +
-                    "                \"connectingType\": \"http://b3mn.org/stencilset/bpmn2.0#SequenceFlow\",\n" +
-                    "                \"parent\" : [\"2\"],\n" +
-                    "                \"children\" : [\"5\"],\n" +
-                    "                \"xyOffset\" : [60,-45]\n" +
-                    "            },\n" +
-                    "            {\n" +
-                    "                \"id\" : \"4\",\n" +
-                    "                \"name\": \"c\",\n" +
-                    "                \"nodetype\" : \"http://b3mn.org/stencilset/bpmn2.0#IntermediateMessageEventCatching\",\n" +
-                    "                \"namespace\" : \"http://b3mn.org/stencilset/bpmn2.0#\",\n" +
-                    "                \"connectingType\": \"http://b3mn.org/stencilset/bpmn2.0#SequenceFlow\",\n" +
-                    "                \"parent\" : [\"2\"],\n" +
-                    "                \"children\" : [\"6\"],\n" +
-                    "                \"xyOffset\" : [60,45]\n" +
-                    "            },\n" +
-                    "            {\n" +
-                    "                \"id\": \"5\",\n" +
-                    "                \"name\": \"B\",\n" +
-                    "                \"nodetype\" : \"http://b3mn.org/stencilset/bpmn2.0#Task\",\n" +
-                    "                \"namespace\" : \"http://b3mn.org/stencilset/bpmn2.0#\",\n" +
-                    "                \"connectingType\" : \"http://b3mn.org/stencilset/bpmn2.0#SequenceFlow\",\n" +
-                    "                \"parent\" : [\"3\"],\n" +
-                    "                \"children\" : [],\n" +
-                    "                \"xyOffset\" : [120,0]\n" +
-                    "            },\n" +
-                    "            {\n" +
-                    "                \"id\": \"6\",\n" +
-                    "                \"name\": \"C\",\n" +
-                    "                \"nodetype\" : \"http://b3mn.org/stencilset/bpmn2.0#Task\",\n" +
-                    "                \"namespace\" : \"http://b3mn.org/stencilset/bpmn2.0#\",\n" +
-                    "                \"connectingType\" : \"http://b3mn.org/stencilset/bpmn2.0#SequenceFlow\",\n" +
-                    "                \"parent\" : [\"4\"],\n" +
-                    "                \"children\" : [],\n" +
-                    "                \"xyOffset\" : [120,0]\n" +
-                    "            }\n" +
-                    "        ]\n" +
-                    "    }\n" +
-                    "]");
+            ioService.write(ioService.get(URI.create(globalDirPath + separator + "backboneformsinclude.fw")),
```

Comment:
I'd prefer to see this extracted in a method (like createEmtyFile) and then call it like
```
Arrays.asList("backbonejsinclude.fw", "cancelbutton.fw", ...)
   .forEach(this::createEmptyFile)
```